plenum/test/monitoring/test_revival_spike_resistant_ema_throughput_measurement.py

```diff
@@ -117,14 +145,13 @@ def test_rsr_ema_tm_after_start_stays_in_faded_while_windows_are_empty(tm_after_
     # [0, 15) - [45, 60)
 
     # [60, 75)
-    throughput = tm.get_throughput(62)
-
+    assert tm.get_throughput(62) is None
     assert tm.state == State.FADED
-    assert throughput == 0
+    assert tm.throughput == 0
 
     assert tm.throughput_before_idle == 0
     assert tm.idle_start_ts == 0
-    assert tm.empty_windows_count == 4
+    assert tm.empty_windows_count == 0
```

Comment:
Why it's 0 now? Should we have a new test or edit this one to get the expected behaviour?

Reply:
Why is value "0" bad? We didn't order any transactions and empty_windows_count is 0.

Reply:
Please add a comment to the code explaining why `empty_windows_count` must be 0 here.

Reply:
Added.